modules/FvwmForm/FvwmForm.c

```diff
@@ -100,7 +100,7 @@ int colorset = -1;
 int itemcolorset = 0;
 
 /* global not exported */
-const struct itimerval itv_100ms = { {0L, 100000L}, {0L, 100000L} };
+const struct itimerval itv_100ms = { {0L, 100000L}, {0L, 100000L} }; /* interval = 100ms, value = 100ms */
```

Comment:
This comment isn't part of timer removal, so should be elided.  It's probably not even worth including as separate patch.